electron/launcher/main-process/ipc-listeners/user-logout.js

```diff
@@ -0,0 +1,33 @@
+const request = require('request-promise');
+const ipc = require('electron').ipcMain;
+
+const config = require('../config.json').DJANGO_SERVER;
```

Comment:
Is there a way to do path includes from project root? Relative references don't usually sit well.

Reply:
Root for electron code is now the `main-process` folder.